Description: Upstream patch for option to install maintainer scripts.
Author: TheAssassin
Origin: upstream, https://github.com/TheAssassin/AppImageLauncher/commit/f62742da42dfafc8978f297d5940443c358ad08d
Last-Update: 2019-01-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: AppImageLauncher-1.0.2/resources/CMakeLists.txt
===================================================================
--- AppImageLauncher-1.0.2.orig/resources/CMakeLists.txt	2019-01-11 10:30:48.000000000 +0000
+++ AppImageLauncher-1.0.2/resources/CMakeLists.txt	2019-01-11 10:37:42.638732953 +0000
@@ -21,3 +21,29 @@
     DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/l10n
     DESTINATION ${CMAKE_INSTALL_DATADIR}/appimagelauncher COMPONENT APPIMAGELAUNCHER
 )
+
+set(INSTALL_MAINTAINER_SCRIPTS OFF CACHE BOOL "")
+
+
+# install maintainer scripts for
+if (INSTALL_MAINTAINER_SCRIPTS)
+    message(STATUS "Installing maintainer scripts")
+
+    set(install_maintainer_scripts_dir ${PROJECT_BINARY_DIR}/cmake/install-maintainer-scripts/)
+
+    configure_file(
+        ${PROJECT_SOURCE_DIR}/resources/install-scripts/post-install.in
+        ${install_maintainer_scripts_dir}/post-install
+        @ONLY
+    )
+    configure_file(
+        ${PROJECT_SOURCE_DIR}/resources/install-scripts/post-uninstall.in
+        ${install_maintainer_scripts_dir}/post-uninstall
+        @ONLY
+    )
+
+    install(
+        FILES ${install_maintainer_scripts_dir}/post-install ${install_maintainer_scripts_dir}/post-uninstall
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/appimagelauncher/maintainer-scripts
+    )
+endif()
